chatmeleon-server/controllers/conversation-controller.ts

```diff
@@ -0,0 +1,158 @@
+import prisma from "../libs/prismadb.js"
+import { Request, Response, NextFunction } from "express";
+
+const getAllConversations = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I think there should be a get function for retrieving Conversation but with paging and limits (e.g only retrieve the first 10 Conversations in the first API call)

Reply:
Also, this is wrong by default, we should only get conversations based on a user's ID, and not every conversation should be retrieved for security purposes